### PR TITLE
https://issues.redhat.com/browse/ACM-6250

### PR DESCRIPTION
**Pull request by faaden123**
Will need to update this.
- Mention of including a stronger emphasis to this note beyond *Note*. 
- https://github.com/stolostron/rhacm-docs/blob/2.8_stage/observability/observability_enable.adoc#enabling-observability

### CODE REVIEW

**Comment by ch-stark** [comment]
LGTM

File: observability/observability_enable.adoc [hunk 1]
```diff
@@ -66,7 +66,9 @@ oc create secret generic multiclusterhub-operator-pull-secret \
     -n open-cluster-management-observability \
     --from-literal=.dockerconfigjson="$DOCKER_CONFIG_JSON" \
     --type=kubernetes.io/dockerconfigjson
+
 ----
+*Note:* In the case that your pull secret is modified, ensure the pull secret in observability namespace is also updated accordingly.
```
Comment (faaden123) [inline observability/observability_enable.adoc]: Potential update following convo with Daniel.

**Comment by dciancio** [comment]
@faaden123  In the "Important" box, I would be specific and mention the "global pull secret" (as this implies the one used for the cluster) and you can link this section in our doc which provides details on how the global pull secret is updated (https://docs.openshift.com/container-platform/4.12/openshift_images/managing_images/using-image-pull-secrets.html#images-update-global-pull-secret_using-image-pull-secrets).  This will provide the user some context as to what we mean by a global pull-secret change.

**Comment by faaden123** [comment]
> https://docs.openshift.com/container-platform/4.12/openshift_images/managing_images/using-image-pull-secrets.html#images-update-global-pull-secret_using-image-pull-secrets

I'm not sure I have access to link this file 

> @faaden123 In the "Important" box, I would be specific and mention the "global pull secret" (as this implies the one used for the cluster) and you can link this section in our doc which provides details on how the global pull secret is updated (https://docs.openshift.com/container-platform/4.12/openshift_images/managing_images/using-image-pull-secrets.html#images-update-global-pull-secret_using-image-pull-secrets). This will provide the user some context as to what we mean by a global pull-secret change.

@dciancio 

> @faaden123 In the "Important" box, I would be specific and mention the "global pull secret" (as this implies the one used for the cluster) and you can link this section in our doc which provides details on how the global pull secret is updated (https://docs.openshift.com/container-platform/4.12/openshift_images/managing_images/using-image-pull-secrets.html#images-update-global-pull-secret_using-image-pull-secrets). This will provide the user some context as to what we mean by a global pull-secret change.

@dciancio Updated this note. If you could take a look.

File: observability/observability_enable.adoc [hunk 2]
```diff
@@ -68,6 +68,8 @@ oc create secret generic multiclusterhub-operator-pull-secret \
     --type=kubernetes.io/dockerconfigjson
 ----
 
+*Important:* In the case that your cluster pull secret is modified, ensure the pull-secret in observability namespace is also updated accordingly.  This allows your original global pull-secret to stay intact. See link:https://docs.openshift.com/container-platform/4.12/openshift_images/managing_images/using-image-pull-secrets.html#images-update-global-pull-secret_using-image-pull-secrets[Updating the global pull secret] for more details
```
Comment (swopebe) [inline observability/observability_enable.adoc]: In the case (way too wordy) 
How else can you word this line? No hyphen in pull secret. Merge sentences to get rid of _this allows_ (also wordy).  Missing punctuation.

**Review by swopebe — DISMISSED** [review]

**Review by swopebe — APPROVED** [review]

**Comment by faaden123** [comment]
Issue has bee fixed
